ui/admin/app/routes/scopes/scope/auth-methods.js

```diff
@@ -35,24 +26,8 @@ export default class ScopesScopeAuthMethodsRoute extends Route {
     if (!this.session.isAuthenticated) this.router.transitionTo('index');
   }
 
-  /**
-   * Load all auth-methods under current scope
-   * @return {Promise[AuthMethodModel]}
-   */
-  async model() {
```

Comment:
Asking for learning purposes of IndexedDB: why we do not need `model()` anymore? is because with IndexedDB the model is already load?

Reply:
@calcaide, good question! The model was not removed it was moved to the `scopes.scope.auth-methods.index` route.

Reply:
Ahhh gotcha, now I realize about that! nice call!